src/main/java/teammates/common/util/Templates.java

```diff
@@ -48,6 +48,14 @@ public static String populateTemplate(String template, String... keyValuePairs)
                 FileHelper.readResourceFile("userEmailTemplate-feedbackSessionPublished.html");
         public static final String USER_FEEDBACK_SUBMISSION_CONFIRMATION =
                 FileHelper.readResourceFile("userEmailTemplate-feedbackSubmissionConfirmation.html");
+        public static final String FRAGMENT_SESSION_LINK_RECOVERY_ACCESS_LINKS =
+                FileHelper.readResourceFile("sessionLinksRecoveryEmailTemplateFragment-feedbackSessionAccessLinks.html");
+        public static final String FRAGMENT_COURSE_LINKS_RECOVERY_ACCESS_LINKS =
```

Comment:
btw i feel this name is rather confusing. It sounds like i am recovering some links for the course. What about FRAGMENT_SESSION_LINKS_RECOVERY_ACCESS_LINKS_BY_COURSE?

And the one above, FRAGMENT_SESSION_LINKS_RECOVERY_ACCESS_LINKS_BY_SESSION?